src/main/resources/morph-hbz01-to-lobid.xml

```diff
@@ -1654,10 +1672,9 @@
 		<!-- #######subjects -->
 		<!-- ####################### -->
 		<!-- ddc links -->
-		<data source="70[05][-b][-12345].a" name="@deweyId">
-			<regexp match="([0-9]{3})" format="http://dewey.info/class/${1}/"/>
-			<sanitizeUrl></sanitizeUrl>
-		</data>
+		<call-macro name="addDeweyNotationAndLabel" field="700[-b][-12345].a"/>
+		<call-macro name="addDeweyNotationAndLabel" field="705?[-12345].a"/>
+		<call-macro name="addDeweyNotationAndLabel" field="705?[-12345].c"/>
```

Comment:
Why not making one rule instead of two? I think this should work fine: `field="705?[-12345].[ac]"/>`

Reply:
I tried this first but then in http://lobid.org/hbz01/HT015455455 subfield a is taken for the notation of subfield c (surprisingly we have no label for `338.76629220904`).

Reply:
Oh. Ok then +1.